wp-content/themes/core/assets/js/src/theme/config/options.js

```diff
@@ -1,4 +1,5 @@
 
 // breakpoint settings
 
-export const MOBILE_BREAKPOINT = 768;
+export const MEDIUM_BREAKPOINT = 768;
```

Comment:
@ryanurban notice I'm adjusting the default breakpoints in our JS to better align with the way we do breakpoints in CSS. This has always been a new-dev pain point.